packages/cardano-services/test/Program/loadHttpServer.test.ts

```diff
@@ -71,7 +72,13 @@ describe('loadHttpServer', () => {
         async () =>
           await loadHttpServer({
             apiUrl,
-            serviceNames: [ServiceNames.StakePool]
+            serviceNames: [
```

Comment:
Also best if this part of the change is a separate `test` commit. The idea is to make atomic changes, and the test here is being improved by starting all of the available services. i.e it's an independent improvement

Reply:
[d850ab4](https://github.com/input-output-hk/cardano-js-sdk/pull/303/commits/d850ab413ed691c2eafedf31fcd8644df4f4ac83)